docs/asset-registry/asset-registry-1.0.0.yaml

```
@@ -0,0 +1,710 @@
+
+    InverterDetails:
+      type: object
+      description: This describes the master schema needed for an inverter asset. To be created in MDMS and validated against 
+      properties:
+        capacity:
+          type: integer
+          description: Capacity of the inverter in KW
+    
+    PanelDetails:
+      type: object
+      description: This describes the master schema needed for a panel asset. To be created in MDMS and validated against 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)